process/subprocesses/ghsci.py

```diff
@@ -1136,45 +1134,59 @@ def _create_open_space_areas(self):
 
         open_space_areas_setup(self.codename)
         return 'Open space areas created.'
+    
+    def _compute_lpugs(self):
+        """Identify large public urban green space for this study region."""
+        from _07_large_public_urban_green_space import large_public_urban_green_space
+
+        large_public_urban_green_space(self.codename)
+        return 'Large public urban green space analysis completed.'
+    
+    def _compute_guhvi(self):
+        """Generate global urban heat vulnerability index."""
+        from _08_global_urban_heat_vulnerability_index import global_urban_heat_vulnerability_index
+
+        global_urban_heat_vulnerability_index(self.codename)
+        return 'Global Urban Heat Vulnerability Index analysis completed.'    
 
     def _create_neighbourhoods(self):
         """Create neighbourhood relations between nodes for this study region."""
-        from _07_locate_origins_destinations import nearest_node_locations
+        from _09_locate_origins_destinations import nearest_node_locations

```

Comment:
As suggested elsewhere, I recommend not modifying the numbering sequence of scripts at all, but instead launch the ee workflow as a final part of _11_neighbourhood_analysis.py after calculating other standard indicators.  So we will want to under all these renamings, which should keep things more similar with current main branch too.